src/GraphQL.Tests/Execution/InputConversionTestsBase.cs

```diff
@@ -63,27 +62,27 @@ public class Child
         [Fact]
         public void converts_small_numbers_to_int()
         {
-            var json = @"{'a': 1}";
-            var inputs = json.ToInputs();
+            var json = @"{""a"": 1}";
+            var inputs = VariablesToInputs(json);
```

Comment:
Good trick with testing both serializers with inheritance. Please add one more test where json is `null`, now all cases pass not `null` json.

Reply:
Yea it seemed like the best solution haha. Bit of a weird situation we find ourselves in with extension methods per serializer implementation.

Reply:
Added test, found bug, win win.

Reply:
I believe that we did everything as conveniently as possible.